src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveReviewsService.kt

```diff
@@ -76,4 +89,21 @@ class IncentiveReviewsService(
 
   private fun calcTypeCount(caseNoteUsage: List<CaseNoteUsage>): Int =
     caseNoteUsage.map { it.numCaseNotes }.fold(0) { acc, next -> acc + next }
+
+  private suspend fun getNextReviewDatesForOffenders(bookingIds: Set<Long>): Map<Long, IepResult> {
```

Comment:
given a person can't appear more than once in offender search results, there's probably no need to convert `bookingIds` from list to set back to list

Reply:
Good point - addressed https://github.com/ministryofjustice/hmpps-incentives-api/pull/237/commits/0f7365b19d018a307509345d6d2bcbfacfa0485e